src/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Activity.Id.MachineName.cs

```diff
@@ -3,11 +3,19 @@
     public partial class Activity
     {
         #region private
-        private string generateUniquePrefix()
```

Comment:
Another stylistic thing is that breaking things into files can be good, but only if you each piece is reasonably substantial, AND each piece is very decoupled (they don't share read-write state, and have a good contract between them).   Maybe separating all the ID stuff from the rest of activity would be good, but separating Random from Machine makes the code worse (in my opinion).     But frankly it is easier still to simply put it in the same file (perhaps all grouped into a single region), and really that is the implementation to beat.